src/patternfly/upgrade-examples/LoginUpgradeExamples/docs/code.md

```diff
@@ -0,0 +1,12 @@
+## Overview
+
+When converting PatternFly 3 components to PatternFly 4 components, you must also take into consideration the layouts and sizings that PatternFly 3 utilized from Bootstrap 3.
+
+## Usage
+
+| PF3 Class | Applied To | PF4 Class | Applied To | Outcome |
+| -- | -- | -- | -- | -- |
+| `.login-pf` | `<div>` | `.pf-c-login` | `<div>` | Creates a login containing content. |
+| `.login-pf-heading` | `<div>` | `.pf-c-login__header` | `.pf-c-login` | Creates the header of a login. |    
+| `.login-pf-body` | `<div>` | `.pf-c-login__body` | `.pf-c-login` | Creates the body of a login. Required. | 
+| `.login-pf-footer` | `<div>` | `.pf-c-login__footer` | `.pf-c-login` | Creates the footer of a login. | 
```

Comment:
I think this should be `.login-pf-page-footer` in pf3, and it applies to `<footer>`, not `<div>`. Also `.pf-c-login__footer` in pf4 applies to `<footer>`, not `.pf-c-login`

Reply:
Class and element tags has been relabeled.